venues/auai.org/UAI/2018/process/officialReviewProcess.js

```diff
@@ -0,0 +1,71 @@
+function(){
+    var or3client = lib.or3client;
+
+    var origNote = or3client.or3request(or3client.notesUrl+'?id='+note.forum, {}, 'GET', token);
+    var list = note.invitation.replace(/_/g,' ').split('/');
+    list.splice(list.indexOf('-',1));
+    var conference = list.join(' ');
+
+    origNote.then(function(result) {
+      var forum = result.notes[0];
+      var note_number = forum.number;
+
+      var reviewers = ['auai.org/UAI/2018/Paper' + note_number + '/Reviewers'];
+      var areachairs = ['auai.org/UAI/2018/Paper' + note_number + '/Area_Chairs'];
+      var reviewersUnsubmittedId = 'auai.org/UAI/2018/Paper' + note_number + '/Reviewers/Unsubmitted';
+      var reviewersSubmittedId = 'auai.org/UAI/2018/Paper' + note_number + '/Reviewers/Submitted';
+      var authors = forum.content.authorids;
+
+      var author_mail = {
+        "groups" : authors,
+        "subject": "Review posted to your submitted paper: \"" + forum.content.title + "\"",
+        "message": "Your submissions to " + conference + " has received an official review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum
+      };
+
+      var reviewer_mail = {
+        "groups" : reviewers,
+        "subject": "Review posted to your assigned paper: \"" + forum.content.title + "\"",
+        "message": "A submission to " + conference + ", for which you are an official reviewer, has received a review. \n\nTitle: " + note.content.title + "\n\nReview: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum
```

Comment:
I remembered to get some feedback asking to see the review content in the email.

Reply:
oops, it's actually there already: note.content.review

Reply:
I'm confused - the review content is already in the email. Am I misunderstanding what you mean? 

"message": "A submission to " + conference + ", for which you are an official reviewer, has received a review. \n\nTitle: " + note.content.title + "\n\nReview: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum

Reply:
Ohhh you are right! I'm sorry.